docs/add-ons/configuration.md

```diff
@@ -136,7 +136,7 @@ The configuration for an add-on is stored in `config.json`.
 | privileged | list | no | Privilege for access to hardware/system. Available access: `NET_ADMIN`, `SYS_ADMIN`, `SYS_RAWIO`, `SYS_TIME`, `SYS_NICE`, `SYS_RESOURCE`, `SYS_PTRACE`, `SYS_MODULE` or `DAC_READ_SEARCH`
 | full_access | bool | no | Give full access to hardware like the privileged mode in docker. Works only for not protected add-ons.
 | apparmor | bool/string | no | Enable or disable AppArmor support. If it is enable, you can also use custom profiles with the name of the profile.
-| map | list | no | List of maps for additional Home Assistant folders. Possible values: `config`, `ssl`, `addons`, `backup` or `share`. Defaults to `ro`, which you can change by adding `:rw` to the end of the name.
+| map | list | no | List of maps for additional Home Assistant folders. Possible values: `config`, `ssl`, `addons`, `backup`, `share` or `media`. Defaults to `ro`, which you can change by adding `:rw` to the end of the name.
```

Comment:
```suggestion
| map | list | no | List of maps for additional Home Assistant folders. Possible values: `config`, `ssl`, `addons`, `backup`, `share` or `media`. Unless otherwise specified, access privileges default to `ro`, you can this by adding `:rw` to the end of the name.
```

Currently (to me at least) it reads as if if you don't specify a file name it defaults to `ro`. Obviously with a little more looking it becomes clear but if someone is just scanning through it might confuse

Reply:
Incorrect, those folders are not keys, those are values.

Reply:
Ok looks good then :)